modules/animationqt/src/animationeditordockwidgetqt.cpp

```diff
@@ -190,6 +192,35 @@ AnimationEditorDockWidgetQt::AnimationEditorDockWidgetQt(
     toolBar->setFloatable(false);
     toolBar->setMovable(false);
     mainWindow_->addToolBar(toolBar);
+
+    // Menu
+    auto menu = mainWindow_->menuBar();
```

Comment:
Is it necessary to have a menu bar with just the file menu? I would think just the toolbar buttons are sufficient. Compare with the python/shader editor.

Reply:
The should only be one menubar for the app. How will this work on Mac where the menubar is app global and not window specific.

Reply:
Ok, I can remove the menubar. I guess the toolbar buttons should be descriptive enough. The only problem is if users think that it saves/loads the entire network since the icons are the same, but I think the context might give enough cues

Reply:
I would think so.